src/main/java/nl/tudelft/cse1110/andy/writer/standard/StandardResultWriter.java

```diff
@@ -286,7 +286,7 @@ private void printTestResults(UnitTestsResult tests) {
             return;
 
         l("\n--- JUnit execution");
-        if (tests.getTestsFound() == 0) {
+        if (tests.getTestsFound() == 0 && !tests.hasFailingMessage()) {
```

Comment:
Two checks on the same `test` object, can we create a method inside this `UnitTestsResult` class, that encapsulates `tests.getTestsFound() == 0 && !tests.hasFailingMessage()`, and we give a nice name to it?

Reply:
Fixed